### PR TITLE
MockSpan fix startMicros to implicit value

### PR DESCRIPTION
**Pull request by pavolloffay**
@adriancole @bensigelman could you please review?

### CODE REVIEW

**Comment by pavolloffay** [comment]
@yurishkuro could you please review?

File: opentracing-mock/src/test/java/io/opentracing/mock/MockTracerTest.java [hunk 1]
```diff
@@ -99,4 +101,19 @@ public void testChildSpan() {
...
         assertEquals(parent.context().traceId(), child.context().traceId());
     }
+
+    @Test
+    public void testStartTimestamp() {
+        MockTracer tracer = new MockTracer();
+        final long startMicros = System.currentTimeMillis() * 1000;
+        {
+            tracer.buildSpan("foo").start().finish();
+        }
```
Comment (yurishkuro) [inline opentracing-mock/src/test/java/io/opentracing/mock/MockTracerTest.java]: why extra scope?

Comment (pavolloffay) [inline opentracing-mock/src/test/java/io/opentracing/mock/MockTracerTest.java]: just minimizing sope. It's used in other tests in this file.

**Comment by pavolloffay** [comment]
@yurishkuro PR updated. It assumes that builder should not be used reused. Start time is set only once, or if `withStartTimestamp(time)` is called.

File: opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 The OpenTracing Authors
+ * Copyright 2016-2017 The OpenTracing Authors
```
Comment (pavolloffay) [inline opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java]: Making license plugin happy.

(PR editing `MockSpan` was created in 2016 and merged in 2017).

**Comment by bensigelman** [comment]
Thanks and sorry again (again) for missing this the first time around.

**Pull request merged**